src/main/java/seedu/anilist/ui/TabOption.java

```diff
@@ -69,7 +67,7 @@ private TabOptions parseTabStatus(String tabString) {
     @Override
     public boolean equals(Object other) {
         return other == this // short circuit if same object
-                || (other instanceof Status // instanceof handles nulls
+                || (other instanceof TabOption // instanceof handles nulls
```

Comment:
Shouldn't this follow the same as the previous test where you assert != null?

Reply:
If you mean the one in `Stats.java` I will be removing that